functions.php

```diff
@@ -120,40 +120,77 @@ function panel_login_fail( $username ) {
     if ( !empty($referrer) && !strstr($referrer,'wp-login') && !strstr($referrer,'wp-admin') ) {
         $referrer = explode('?', $referrer);
         parse_str($referrer[1], $referrer_args);
-        unset($referrer_args['idilogout']); 	//unset old idilogout arg if set
-        $referrer_args['idilogin'] = "failed"; 	//let theme know there are errors
+        unset($referrer_args['idilogout']);     //unset old idilogout arg if set
+        $referrer_args['idilogin'] = "failed";     //let theme know there are errors
```

Comment:
There's no strict convention. Their [styleguide](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) says:

"Use single and double quotes when appropriate. If you’re not evaluating anything in the string, use single quotes."

In this case I'll change it to single quotes to be consistent with the usage so far.